andy/exercise_2.py

```diff
@@ -0,0 +1,60 @@
+"""" Exercise 2 """
+# test 1
+# Create tuple with chars from a long string.
+TEST_STRING = "PeterPidorButNoOnePaysAttention"
+TUPLE_RESULT = tuple(TEST_STRING)
+# Slice last 20 chars in reversed order beging from 5th (5th from end).
+SLICED = TUPLE_RESULT[len(TUPLE_RESULT) - 4:len(TUPLE_RESULT) - 20:-1]
+# Create dictionary from sliced chars. Print sorted keys of this dictionary.
+KEY_VALUE_TUPLE = tuple(SLICED[i: i + 2] for i in range(0, len(SLICED), 2))
+print("Test 1")
+print("Sorted keys: ", sorted(dict(KEY_VALUE_TUPLE).keys()))
+
+# test 2
+# Create list of chars from a long string.
+FIRST_LIST = list(TEST_STRING)
+# create another string
+NEW_STRING = "AndHomaPidorToo"
+# Extend list by chars from another string.
+FIRST_LIST.extend(list(NEW_STRING))
+# Remove duplicated items form list.
+FIRST_LIST = list(set(FIRST_LIST))
+# Delete middle element using slice functionality (important).
+print("Test 2")
+print("Before removing: ", FIRST_LIST)
+MIDDLE_ELEMENT = FIRST_LIST[int(len(FIRST_LIST) / 2):int(len(FIRST_LIST) / 2 + 1)]
+FIRST_LIST.remove(MIDDLE_ELEMENT[0])
```

Comment:
in this case, it would be better to use del FIRST_LIST[index], because 'remove' method returns removed value